code/game/objects/items/control_wand.dm

```diff
@@ -29,6 +29,8 @@
 	to_chat(user, "Now in mode: [mode].")
 
 /obj/item/weapon/door_remote/afterattack(obj/machinery/door/airlock/D, mob/user)
+	if((istype(D, /obj/machinery/door/airlock/hatch/gamma)) && !(seclevel2num(get_security_level()) == SEC_LEVEL_GAMMA))
```

Comment:
This is snowflakey as fuck, just make a door variable to prevent access via door remotes. As specific as that is, it might also be useful for away missions and such.